rules/autoconsent/complianz-optin.json

```
@@ -12,7 +12,8 @@
         { "click": ".cc-settings" },
         { "waitForVisible": "[aria-label=\"cookies preferences popup\"]" },
         { "click": "[aria-label=\"cookies preferences popup\"] input[type=checkbox]:not([disabled])", "all": true },
```

```suggestion
        { "click": "[aria-label=\"cookies preferences popup\"] input[type=checkbox]:not([disabled]):checked", "all": true, "optional": true },
```
Let's only click checked boxes (this opted me into some stuff when I tested it).